src/Quartz.HttpClient/HttpClient/Extensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Quartz.Impl.Matchers;
+using Quartz.Util;
+
+namespace Quartz.HttpClient;
+
+internal static class Extensions
+{
+    public static string ToUrlParameters<T>(this GroupMatcher<T> groupMatcher) where T : Key<T>
+    {
+        if (groupMatcher == null || string.IsNullOrWhiteSpace(groupMatcher.CompareToValue))
+        {
+            return "";
+        }
+
+        if (groupMatcher.CompareWithOperator.Equals(StringOperator.Equality))
+        {
+            return $"groupEquals={groupMatcher.CompareToValue}";
```

Comment:
probably not worth the change, but this could maybe also be `group=*{input}`, probably forbidden by URI schema

Reply:
Asterisk seems to be reserved character in urls, so might not make things easier.